README.md

```diff
@@ -30,33 +30,29 @@ Clone the repository.
 
 | Directory  | Chapter |
 | ------------- | ------------- |
-| Chapter 1 - Introduction | src/main/scala/com/professionalscala/ch1 |
-| Chapter 2 - Functional Programming | src/main/scala/com/professionalscala/ch2 |
-| Chapter 3 - Concurrency | src/main/scala/com/professionalscala/ch3 |
-| Chapter 3 - Scala and Java Collections | src/main/scala/com/wrox/professionalscala/ch03 |
+| Chapter 1 - Introduction | src/main/scala/com/wrox/professionalscala/ch01 |
```

Comment:
This change should not be merged because it fails to describe the project structure here: https://github.com/Mironor/professional-scala-code/tree/master/src/main/scala/com/professionalscala. Do not assume this table describes only one repository.